frontend/components/Blocks/CardsBlock/CardBlock.module.css

```diff
@@ -0,0 +1,8 @@
+.Image {
+  object-fit: cover;
+  object-position: center;
+  height: 100%;
+  width: 100%;
+  max-width: none;
+  max-height: none;
+}
```

Comment:
Is this class used anywhere? I think this is all done directly in the Card component using tailwind classes. If so, this can also be removed.

Reply:
Nice finding, I removed the obsolete file cardblock.module.css